### PR TITLE
ontology url

### PR DESCRIPTION
**Pull request by Ahmedreyadh1985sh**
Hello sir
I changed the routing address
Documentation is available at: https://ahmedreyadh1985sh.github.io/GVTPDonto/Documentation/index-en.html

I need this URI https://w3id.org/GVTPDonto redirects to https://ahmedreyadh1985sh.github.io/GVTPDonto/Documentation/index-en.html

Contact: https://github.com/Ahmedreyadh1985sh
         ahmed.riyadh.shaheen@gmail.com

### CODE REVIEW

File: GVTPDOnto/README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 
-Documentation is available at: https://ahmedreyadh1985sh.github.io/GVTPDonto/OntoDoc/GVTPDOnt/index-en.html
+Documentation is available at: https://ahmedreyadh1985sh.github.io/GVTPDonto/Documentation/index-en.html
 
-I need this URI https://w3id.org/GVTPDOnt redirects to https://ahmedreyadh1985sh.github.io/GVTPDonto/OntoDoc/GVTPDOnt/index-en.html
+I need this URI https://w3id.org/GVTPDOnto redirects to https://ahmedreyadh1985sh.github.io/GVTPDonto/Documentation/index-en.html
```
Comment (TallTed) [inline GVTPDOnto/README.md:4]:
```suggestion
This URI https://w3id.org/GVTPDOnto redirects to https://ahmedreyadh1985sh.github.io/GVTPDonto/Documentation/index-en.html

```

File: GVTPDOnto/README.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 
-Documentation is available at: https://ahmedreyadh1985sh.github.io/GVTPDonto/OntoDoc/GVTPDOnt/index-en.html
+Documentation is available at: https://ahmedreyadh1985sh.github.io/GVTPDonto/Documentation/index-en.html
 
-I need this URI https://w3id.org/GVTPDOnt redirects to https://ahmedreyadh1985sh.github.io/GVTPDonto/OntoDoc/GVTPDOnt/index-en.html
+I need this URI https://w3id.org/GVTPDOnto redirects to https://ahmedreyadh1985sh.github.io/GVTPDonto/Documentation/index-en.html
 
 Contact: https://github.com/Ahmedreyadh1985sh
          ahmed.riyadh.shaheen@gmail.com
```
Comment (TallTed) [inline GVTPDOnto/README.md:7]:
```suggestion
Contact: 
* https://github.com/Ahmedreyadh1985sh
* ahmed.riyadh.shaheen@gmail.com

```

**Review by TallTed — CHANGES_REQUESTED** [review]

**Comment by dgarijo** [comment]
@Ahmedreyadh1985sh you are still changing the root htaccess of w3id, which is not acceptable :(
I suggest closing this PR and doing your changes only in the folders you want to declare, without touching the main `.htaccess` file

File: .htaccess [hunk 3]
```diff
@@ -1,8 +1,74 @@
...
+RewriteRule ^(.+)$ https://ahmedreyadh1985sh.github.io/GVTPDonto/Documentation/ontology.ttl [R=303,L]
+
+
+
+RewriteCond %{HTTP_ACCEPT} .+
+RewriteRule ^(.*)$ https://ahmedreyadh1985sh.github.io/GVTPDonto/Documentation/406.html [R=406,L]
+# Default response
+# ---------------------------
+# Rewrite rule to serve the RDF/XML content from the vocabulary URI by default
+RewriteRule ^$ https://ahmedreyadh1985sh.github.io/GVTPDonto/Documentation/ontology.owl [R=303,L]
```
Comment (dgarijo) [inline .htaccess:74]: This is the file that you need NOT edit

**Comment by davidlehn** [comment]
Also please do not have two directories with names that only differ in case.

**Pull request closed without merging**